integration/test/basic/error.go

```diff
@@ -0,0 +1,9 @@
+// +build k8srequired
+
+package basic
+
+import "github.com/giantswarm/microerror"
+
+var executionFailedError = &microerror.Error{
+	Kind: "executiomFailedError",
```

Comment:
Check the repo where you keep copying this :joy: 

```suggestion
	Kind: "executionFailedError",
```

Reply:
Executiom is the new execution. 😂 

But I found the source. https://github.com/giantswarm/metrics-server-app/pull/38